pkg/apis/autoscaling/v1/fleetautoscaler.go

```diff
@@ -195,6 +210,61 @@ type ListPolicy struct {
 	BufferSize intstr.IntOrString `json:"bufferSize"`
 }
 
+// Between defines the time period that the policy is eligible to be applied.
+type Between struct {
+	// Start is the datetime that the policy is eligible to be applied.
+	// This field must conform to RFC3339 format. If this field not set or is in the past, the policy is eligible to be applied
+	// as soon as the fleet autoscaler is running.
+	Start string `json:"start"`
+
+	// End is the datetime that the policy is no longer eligible to be applied.
+	// This field must conform to RFC3339 format. If not set, the policy is always eligible to be applied, after the start time above.
+	End string `json:"end"`
+}
+
+// ActivePeriod defines the time period that the policy is applied.
+type ActivePeriod struct {
+	// Timezone to be used for the startCron field. If not set, startCron is defaulted to the UTC timezone.
+	Timezone string `json:"timezone"`
```

Comment:
Same here, if the types we want to use have appropriate JSON interface arguments, we can likely use them directly.

Reply:
I could not find one for `time.Location`, so I've kept it as a string for now. However, I was able to find/use one for the `Start` and `End` fields.